app-modules/task/resources/views/filament/pages/list-tasks.blade.php

```diff
@@ -0,0 +1,117 @@
+@php
+    use Assist\Livewire\Task\TaskKanban;use Filament\Support\Facades\FilamentView;
+@endphp
+<x-filament-panels::page
+        @class([
+            'fi-resource-list-records-page',
+            'fi-resource-' . str_replace('/', '-', $this->getResource()::getSlug()),
+        ])
+>
+    <div class="w-full flex justify-start">
+        <div class="grid max-w-xs grid-cols-2 gap-1 p-1 bg-gray-100 rounded-lg dark:bg-gray-800" role="group">
+            <button
+                    type="button"
+                    @class([
+                        'px-5 py-1.5 text-xs font-medium rounded-lg',
+                        'text-white bg-gray-900 dark:bg-gray-300 dark:text-gray-900' => $viewType === 'table',
+                        'text-gray-900 hover:bg-gray-200 dark:text-white dark:hover:bg-gray-700' => $viewType !== 'table',
+                    ])
+                    wire:click="setViewType('table')"
+            >
+                <x-filament::icon class="w-6 h-6" icon="heroicon-m-table-cells"/>
+            </button>
+            <button
+                    type="button"
+                    @class([
+                        'px-5 py-1.5 text-xs font-medium rounded-lg',
+                        'text-white bg-gray-900 dark:bg-gray-300 dark:text-gray-900' => $viewType === 'kanban',
+                        'text-gray-900 hover:bg-gray-200 dark:text-white dark:hover:bg-gray-700' => $viewType !== 'kanban',
+                    ])
+                    wire:click="setViewType('kanban')"
+            >
+                <x-filament::icon class="w-6 h-6" icon="heroicon-m-view-columns"/>
+            </button>
+        </div>
+    </div>
+    @if($viewType === 'table')
+        <div class="flex flex-col gap-y-6">
+            @if (count($tabs = $this->getTabs()))
+                <x-filament::tabs>
+                    {{ FilamentView::renderHook('panels::resource.pages.list-records.tabs.start', scopes: $this->getRenderHookScopes()) }}
+
+                    @foreach ($tabs as $tabKey => $tab)
+                        @php
+                            $activeTab = strval($activeTab);
+                            $tabKey = strval($tabKey);
+                        @endphp
+
+                        <x-filament::tabs.item
+                                :active="$activeTab === $tabKey"
+                                :badge="$tab->getBadge()"
+                                :icon="$tab->getIcon()"
+                                :icon-position="$tab->getIconPosition()"
+                                :wire:click="'$set(\'activeTab\', ' . (filled($tabKey) ? ('\'' . $tabKey . '\'') : 'null') . ')'"
```

Comment:
Weird indentation, if we swap out the Blade formatter (for the better one https://github.com/Stillat/blade-parser-typescript/tree/main/formatter) it should handle this

Reply:
I believe I have fixed the indentation. But I would consider changing the Blade formatted outside of the scope here. We can create a separate ticket for that.